lib/gollum-lib/markups.rb

```diff
@@ -10,6 +9,13 @@ def gem_exists?(name)
       Gem::Specification.find {|spec| spec.name == name} != nil
     end
 
+    def gems_exist?(names)
```

Comment:
Though this method behaves idiomatically (question mark indicating a boolean return value), based on its name, I would actually expect this method to return an array of booleans (one for each gem). Instead, it returns false if a single gem does not exist. I would prefer to rename this method to `all_gems_available?`.